dispatchrouter/pom.xml

```diff
@@ -94,7 +94,7 @@
               <images>
                 <image>
                   <build>
-                    <from>eclipsehono/qpid-dispatch:${qpid-dispatch.version}</from>
+                    <from>enmasseproject/qdrouterd:0.10.0</from>
```

Comment:
These images are tagged to enmasse releases. It's still router 0.8.0. The previous match was a conicidence